yarn.lock

```diff
@@ -14434,7 +14434,7 @@ typedarray@^0.0.6:
   resolved "https://registry.yarnpkg.com/typedarray/-/typedarray-0.0.6.tgz#867ac74e3864187b1d3d47d996a78ec5c8830777"
   integrity sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=
 
-typescript@^4.1.2:
+typescript@4.5.5:
```

Comment:
had to make sure we have a fixed version, because of weird bugs with different once

Reply:
perfect!